src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -66,7 +83,7 @@ public class TestSaga08 : Saga<TestSaga08.SagaData08>,
 
                 public Task Handle(StartSagaMessage message, IMessageHandlerContext context)
                 {
-                    return context.SendLocal(new SomeOtherMessage());
+                    return context.SendLocal(new StartSagaMessageByHandler());
```

Comment:
after thinking this a bit more through, I don't think we need that extra indirection. Instead set the SagaId on the message property and change the finder to use the messages sagaid to fetch the existing saga id from the storage. I don't really see why we need to manually create a new saga instance.